modules/nextflow/src/test/groovy/nextflow/processor/TaskHandlerTest.groovy

```diff
@@ -96,7 +97,7 @@ class TaskHandlerTest extends Specification {
         trace.read_bytes == 20
         trace.write_bytes == 30
         trace.queue == 'longjobs'
-        trace.cpus == 2
+        trace.cpus == CpuUnit.of('2100m').toCores()
```

Comment:
Not sure if the trace record should always round up to the next core. Seems like it should depend on the executor, i.e. whether the executor rounds up or not.

Reply:
If we do change the trace record, we'll also need to change the `cpus` task metric in Tower to `DECIMAL`

Reply:
For simplicity and backwards compatibility with Tower, I think rounding here is fine. Tower uses `%cpu` anyway for the resource optimization, so it won't be affected.